src/platform/site-wide/side-nav/components/DuplicateLineLabel.js

```diff
@@ -35,19 +31,6 @@ const DuplicateLineLabel = ({ depth, item }) => {
   return (
     <>
       <div className="line" />
```

Comment:
This div should be remove too, I think

Reply:
that's still needed it's the line
<img width="391" alt="Screen Shot 2020-03-24 at 1 06 04 PM" src="https://user-images.githubusercontent.com/100468/77461243-42d37180-6dd0-11ea-8ff0-80c97e74857a.png">